helmfile.d/values/_helpers.tpl

```diff
@@ -83,3 +83,11 @@ after the separator.
 {{- define "gen.reg-rep-img" -}}
 {{ join "/" (compact (list .registry .repository .image)) }}
 {{- end }}
+
+{{- define "gen.rep-img" -}}
```

Comment:
Question: Why do we need this? It isn't used anywhere from what I can see

Reply:
This seem to serve no purpose here. I'll just remove it